server/controllers/stock/reports/purchaseOrderAnalysis/report.handlebars

```diff
@@ -11,51 +11,52 @@
         <!-- page title  -->
         <h3 class="text-center text-bold text-uppercase">{{translate 'REPORT.PURCHASE_ORDER_ANALYSIS.TITLE'}}</h3>
         <br>
-        <table style="width: 90%; margin-right: auto; margin-left: auto" class="table table-condensed table-report table-bordered">
+        <table class="table-condensed table-report">
```

Comment:
You still need the class "table" here, I think.  It doesn't render on the whole page.

Reply:
The idea was to have a table width defined by the table content and not a table with 100% width

Reply:
I think that makes sense for viewing, but not for printing.  On printing, we want to have a predictable layout for each report.  For example, page numbers should always be in the bottom left, the enterprise name should always be in the top left, etc...